schema.prisma

```diff
@@ -86,14 +86,28 @@ model Company {
   legalPersonName String?       @map("legal_person_name")
   countryCode     String?       @map("country_code") @db.Citext
   cityId          String?       @map("city_id") @db.Uuid
+  personId        String?       @unique @map("person_id") @db.Uuid
```

Comment:
With this change we are saying that one or many companies can be associated with one person and this could be quite restrictive.
It looks more natural for the connection to be many people to be associated with one company. So the suggestion is to put companyId in the Person table.